src/plugins/cache.js

```diff
@@ -107,6 +127,21 @@ export default class Cache {
 
   async fetchManagers (...managers: Manager[]): Promise<Plugin[]> {
     let plugins = []
+    if (this.cache.node_version !== process.version) {
+      let lock = new Lock(this.out)
+      let handled = true
+
+      let downgrade = await lock.upgrade()
+      for (let manager of managers) {
+        handled = await manager.handleNodeVersionChange() && handled
```

Comment:
@dickeyxxx what do you want to happen if the rebuild fails?  I included this logic so that if the rebuild failed that it would attempt to rebuild again next time a command is run.  If you think it is better to ignore build failures and not try to automatically rebuild, I am okay with that since we do dump plugin loading issues.